packages/dev/test-utils/package.json

```diff
@@ -32,6 +32,7 @@
     "url": "https://github.com/adobe/react-spectrum"
   },
   "dependencies": {
+    "@react-aria/dnd": "^3.0.1",
```

Comment:
this is likely to be a circular dependency
probably better to either include the test utils with the aria package, or ship a separate companion package

Reply:
I actually don't think this is needed, this might be left over from something I accidentally imported.